release-schema.json

```diff
@@ -1,5 +1,22 @@
 {
   "definitions": {
+    "Tender": {
+      "properties": {
+        "hasRenewal": {
+          "title": "Has renewal",
+          "description": "Whether the future contracts can be renewed.",
```

Comment:
Do we need the word "future" in both descriptions? We don't use it for Lot. From the moment of the contract notice, the contracts are still in the future, but from the perspective of an old contracting process, they are not. I would prefer removing it.

Reply:
Done, I've removed both in 9cf5d6d

Reply:
Thanks! If you want to save time, you can just commit the change and re-request a review. I will assume you addressed the comments, and I can click "View changes" to see what's new :)